package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "phantomjs",
-  "version": "1.9.19",
+  "version": "2.1.1",
```

Comment:
My apologies, I misunderstood the used process. I get it now :)
Thanks @mgol :)